apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/ShardAddConstraintsSchemaVersion.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright © 2013-2016 The Nxt Core Developers.
+ * Copyright © 2016-2017 Jelurida IP B.V.
+ *
+ * See the LICENSE.txt file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with Jelurida B.V.,
+ * no part of the Nxt software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE.txt file.
+ * 
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+/*
+ * Copyright © 2018 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.core.db;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+import javax.sql.DataSource;
+
+import org.slf4j.Logger;
+
+public class ShardAddConstraintsSchemaVersion extends ShardInitTableSchemaVersion {
+    private static final Logger log = getLogger(ShardAddConstraintsSchemaVersion.class);
+
+    protected int update(int nextUpdate) {
+        nextUpdate = super.update(nextUpdate);
+        switch (nextUpdate) {
+/*  ---------------------- BLOCK -------------------    */
+            case 6:
```

Comment:
I'm not sure I've got what do you mean.

Reply:
When you create ShardAddConstraintsSchemaVersion, pass to the constructor `startVersion`, from which you will start to apply sql updates instead of using case with hardcoded literals such as `6 7 8 9 10`.